resources/data/darwin_deps-0.9.3.json

```
@@ -78,6 +78,43 @@
                 }
             ]
         },
+        {
+            "type": "Checkable",
+            "steps": [
+                {
+                    "type": "Step",
+                    "title": "Install GN build system",
+                    "description": [
+                        "_Note: If you are not interested in the [Connected Home over IP](https://www.connectedhomeip.com/) applications, you can omit steps presented in this section._",
+                        "Download and extract [GN](https://gn.googlesource.com/gn/) build system executable archive used by the Connected Home over IP to generate output files:",
+                        {
+                            "type": "commands",
+                            "description": [
+                                "mkdir ${HOME}/gn && cd ${HOME}/gn",
+                                "wget -O gn.zip https://chrome-infra-packages.appspot.com/dl/gn/gn/mac-amd64/+/latest",
+                                "unzip gn.zip",
+                                "rm gn.zip"
+                            ]
+                        },
+                        "Put GN location into the system PATH:",
```

```suggestion
                        "Add the GN location to the system PATH:",
```
Shouldn't this be a proper step, introduced with a number?

---

Done